lib/ZumoOta/src/FileManager.cpp

```diff
@@ -80,3 +80,38 @@ bool FileManager::init()
     }
     return true;
 }
+
+size_t FileManager::readBytes(const char* firmwareName, uint8_t* buffer)

```

Comment:
This is broken in several ways.

 * a read function does't open /close files, it operates on an open file handle
 * you need to pass the number of bytes to read, and return what you really got
 * Read function doesn't seek, it just keeps reading the file handle.

How do you detect when you read all bytes  of the bytes.

Reply:
It will be adjust  by the next commit.